robot/client/client_test.go

```diff
@@ -1988,8 +2063,9 @@ func TestMachineStatus(t *testing.T) {
 						Revision: "rev0",
 					},
 				},
+				State: robot.StateRunning,
 			},
-			1,
+			2, // once for client.New call and once for MachineStatus call
```

Comment:
My comment here and below is hopefully at least somewhat self-explanatory, but the idea here is that this test calls `client.New` and _then_ `rc.MachineStatus` to assert on the structure of the machine status. Since `client.New` now calls `MachineStatus` in testing environments, the count here and below is increased by a factor of two.